src/redux/categories/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore, combineReducers } from 'redux';
+import bookReducer from '../books/books';
+import checkStatusReducer from './categories';
+
+const reducer = combineReducers({
+  bookReducer,
+  checkStatusReducer,
+});
+
+const store = configureStore(reducer);
+
```

Comment:
- [ ] Please change the function `configureStore` with `createStore` function and also import `createStore` from redux. `configureStore` is not part of redux, This function belongs to the redux toolkit if using the code above you pass your store to component your application will not work.

Reply:
@thecodechaser Thank you for the feedback. Can you please give me more details about the creatStore function?
because I follow the project requirement to the best of my knowledge.

Reply:
@principles31, `createStore` is a redux function to create a store. You can create store using configureStore but needs to be imported from redux-toolkit. You can read more [Here](https://redux-toolkit.js.org/api/configureStore)

Reply:
@thecodechaser, Please I'm still getting confused about how  to implement that and in which line of the configureStore.js file

Reply:
@principles31, Just replace `configureStore` with `createStore` on line 1 and line 10.

Reply:
@thecodechaser, Thanks for the effort and the step-by-step you took me through to make the project work as required.
The above changes have been implemented for your review.Thank you